sql/hive/src/main/scala/org/apache/spark/sql/hive/client/HiveShim.scala

```diff
@@ -1073,7 +1073,7 @@ private[client] class Shim_v2_0 extends Shim_v1_2 {
 private[client] class Shim_v2_1 extends Shim_v2_0 {
 
   // true if there is any following stats task
-  protected lazy val hasFollowingStatsTask = JBoolean.FALSE
+  protected lazy val hasFollowingStatsTask = Boolean.box(SQLConf.get.autoSizeUpdateEnabled)
```

Comment:
Is it possible that we can add a test case in org/apache/spark/sql/hive/client/VersionsSuite.scala?

Reply:
Sorry. Hive 1 also has this problem. We need a common way.

Reply:
Yes. This first approach is a wrong way. Thank you for pivoting.